pipeline/tasks/veneer_tasks.py

```diff
@@ -23,6 +23,50 @@
 from pipeline.utils import lang_params
 
 
+class VeneerConfigGenTask(task_base.TaskBase):
+    """Generates Veneer config file"""
+    default_provides = 'intermediate_config_location'
+
+    def execute(self, gapi_tools_path, descriptor_set, output_dir, api_name):
+        config_gen_dir = os.path.join(output_dir, api_name + '-config-gen')
+        subprocess.check_call(['mkdir', '-p', config_gen_dir])
+        config_gen_path = os.path.join(config_gen_dir,
+                                       api_name + '_veneer.yaml')
+        args = [
+            '--descriptor_set=' + os.path.abspath(descriptor_set),
+            '--output=' + os.path.abspath(config_gen_path)
+            ]
+        clargs = '-Pclargs=' + ','.join(args)
+        subprocess.check_call([os.path.join(gapi_tools_path, 'gradlew'),
+                               '-p', gapi_tools_path, 'runConfigGen', clargs])
+
+        return config_gen_path
+
+    def validate(self):
+        return [vgen_requirements.ConfigGenRequirements]
+
+
+class VeneerConfigMoveTask(task_base.TaskBase):
+    """Move config file to veneer_api_yaml location"""
+
+    def execute(self, intermediate_config_location, veneer_api_yaml):
+        for conf_out in veneer_api_yaml:
```

Comment:
I think we should fail the process - we can't produce what the user is expecting in this case.